tutorials/source-modeling/plot_beamformer_lcmv.py

```diff
@@ -0,0 +1,169 @@
+data_cov.plot(epochs.info)
+
+###############################################################################
+# Compute the spatial filter
+# --------------------------
+# Now we can compute the spatial filter.
+# When looking at the covariance matrix plots, we can see that our data is
+# slightly rank-deficient. Thus, we will regularize the covariance matrix by
+# setting the parameter ``reg=0.05``. This corresponds to loading the
+# diagonal of the covariance matrix with 5% of the sensor power.
+#
+# Different variants of the LCMV beamformer exist. We will compute a
+# unit-noise-gain beamformer, which normalizes the beamformer weights to take
+# care of an inherent depth bias. To achieve this, we set
+# ``weight_norm='unit-noise-gain'``.
+# This parameter can also be set to ``'nai'`` (Neural Activity Index, see [1]_)
+# which implements
+# a further normalization with the estimated noise. An alternative way to take
+# care of this depth bias is to use the ``depth`` parameter and normalize the
+# forward solution instead of the weights. Note that if you compare conditions,
+# the depth bias will cancel out and it is possible to set both parameters to
+# ``None``.
+#
+# Furthermore, we will optimize the orientation of the sources such that output
+# power is maximized. This is achieved by setting ``pick_ori='max-power'``.
+# This gives us one source estimate per source (i.e., voxel), which is known
+# as a scalar beamformer. It is also possible to compute a vector beamformer,
+# which gives back three estimates per voxel, corresponding to the three
+# directions of the source. This can be achieved by setting
+# ``pick_ori='vector'``.
+
+filters = make_lcmv(evoked.info, forward, data_cov, reg=0.05,
+                    noise_cov=noise_cov, pick_ori='max-power',
+                    weight_norm='unit-noise-gain', rank=None)
+
+# You can save the filter for later use with:
+# filters.save('filters-lcmv.h5')
+
+###############################################################################
+# Apply the spatial filter
+# ------------------------
+# The spatial filter can be applied to different data types: raw, epochs,
+# evoked data or the data covariance matrix to gain a static image of power.
+# The function to apply the spatial filter to :class:`~mne.Evoked` data is
+# :func:`~mne.beamformer.apply_lcmv` which is
+# what we will use here. The other functions are
+# :func:`~mne.beamformer.apply_lcmv_raw`,
+# :func:`~mne.beamformer.apply_lcmv_epochs`, and
+# :func:`~mne.beamformer.apply_lcmv_cov`.
+
+stc = apply_lcmv(evoked, filters, max_ori_out='signed')
+
+###############################################################################
+# Visualize the reconstructed source activity
+# -------------------------------------------
+
+lims = [0.3, 0.45, 0.6]
+
+stc.plot(
+    src=forward['src'], subject='sample', subjects_dir=subjects_dir,
+    clim=dict(kind='value', pos_lims=lims), mode='stat_map',
+    initial_time=0.087, verbose=True)
+stc.plot(
```

Comment:
if you separate the two calls to `stc.plot` with a single line of 79 `#` marks, it will plot them above/below (and larger) rather than side by side.  I think that's worth it for these plots.

Reply:
Done.